tests/rms/test_create_rft_ertobs.py

```diff
@@ -157,6 +162,19 @@ def test_get_well_coords():
         == np.array([[0, 0, 0, 0], [2000, 0, 0, 2000]])
     ).all()
 
+    # Alternative trajectory name in RMS:
+    assert (
+        create_rft_ertobs.get_well_coords(
+            rms_project_mock, "R-99", trajectory_name="Imported trajectory"
+        )
+        == np.array([[0, 0, 0, 0], [2000, 0, 0, 2000]])
+    ).all()
```

Comment:
Should it not be an error message sent in case assert test is False ?

Reply:
Did not understand? This is not an error condition, so no error message.